pkg/proto/datadog/remoteconfig/remoteconfig.proto

```diff
@@ -45,6 +45,8 @@ message LatestConfigsRequest {
   // timestamp and snapshot versions move in tandem so they are the same.
```

Comment:
why is this? If Fleet Signer signs snapshots and timestamp we can have short-lived timestamps and long-lived snapshots

Reply:
Will answer internally.